config/honeybadger.yml

```diff
@@ -1 +1 @@
-api_key: <%= Rails.application.credentials.dig(:honeybadger, :api_key) %>
+api_key: "<%= Rails.application.credentials.dig(:honeybadger, :api_key) %>"
```

Comment:
What's the point of this change?

Reply:
I think it happened when I was adding additional config for grouping LinkedIn API errors as per earlier comment.. Will revert this change.